### PR TITLE
New cm

### PR DESCRIPTION
**Pull request by evisochek**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://github.com/demisto/etc/issues/40375),https://github.com/demisto/etc/issues/43880

## Description
The firewall change management is a complex and delicate process that needs to be performed with extreme caution due to the consequences that can be caused due to a mistake,
If you use Pan-Os or Panorama as your enterprise firewall and Jira or ServiceNow as your enterprise ticking this pack will assist you perform a well coordinated and documented process.

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [x] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [x] Documentation

### CODE REVIEW

**Comment by ShirleyDenkberg** [comment]
@evisochek - Please note the following:
- The Packs/Change_Management/Scripts/IncidentState/Readme has not content.
- The Packs/Change_Management/Readme file needs a link to the article.

**Comment by ShirleyDenkberg** [comment]
@MosheEichler @barchen1 @iyeshaya Doc review completed.

**Review by David-BMS — APPROVED** [review]
Please generate new images, as the existing ones are not up to date

Make sure new content is for for 6.1.0

For existing fields you can associate to all

File: Packs/Change_Management/Playbooks/playbook-Jira_Change_Management.yml [hunk 1]
```diff
@@ -0,0 +1,1269 @@
...
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "25":
+    id: "25"
+    taskid: 8681dac6-aef5-4808-8154-f8127ae24055
+    type: regular
+    task:
+      id: 8681dac6-aef5-4808-8154-f8127ae24055
+      version: -1
+      name: export to CSV
```
[the remark this answers is not in the document]
Comment (evisochek) [reply Packs/Change_Management/Playbooks/playbook-Jira_Change_Management.yml]: Fixed

**Pull request merged**